Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM openjdk:8-jre-alpine
```

Comment:
I think the alpine image seems not maintained. (it's updated two years ago)
https://hub.docker.com/_/openjdk?tab=tags&page=1&name=8-jre-alpine
So, we should use another image.